components/engine_traits/src/cf_defs.rs

```diff
@@ -14,11 +14,5 @@ pub fn name_to_cf(name: &str) -> Option<CfName> {
     if name.is_empty() {
         return Some(CF_DEFAULT);
     }
-    for c in ALL_CFS {
-        if name == *c {
-            return Some(c);
-        }
-    }
-
-    None
+    ALL_CFS.iter().find(|&c| name == *c).copied()
```

Comment:
`copied` can also be used in this case.

Reply:
Which case are you referring to? This LoC already uses `copied`.

Reply:
I think what BusyJay expects is like `ALL_CFS.iter().copied().find(|c| name == *c)`, so there is no reference matching.

Reply:
Oh, right.